app/src/main/res/layout/camera.xml

```diff
@@ -14,23 +14,21 @@
             android:id="@+id/lnCam"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:gravity="center" >
+            android:layout_marginTop="16dp">
 
             <ImageButton
                 android:id="@+id/switch_camera"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_horizontal|center"
+                android:layout_width="20dp"
```

Comment:
Just giving small hardcoded value will not solve the problem.
Best is if you set it programmatically according to the screen size.
Also always add any dimension in `dimen.xml`